keyboards/splitkb/kyria/keymaps/cameronjlarsen/rules.mk

```
@@ -0,0 +1,17 @@
+SRC += features/oneshot.c
```

```suggestion
SRC += $(USER_PATH)features/oneshot.c
```

Or you can handle this mostly at the userspace level. Eg, add a rules.mk there with something like:
```make
CUSTOM_ONESHOT_ENABLE ?= no
ifeq ($(strip $(CUSTOM_ONESHOT_ENABLE)), yes)
    SRC += $(USER_PATH)/features/oneshot.c
    OPT_DEFS += -DCUSTOM_ONESHOT_ENABLE
endif
```

This would allow you to add just `CUSTOM_ONESHOT_ENABLE = yes` to your keymap's rules.mk, instead.